### DISCUSSION TITLE
How does this project build on or differ from Advanced Slides?

### DISCUSSION THREAD

**Discussion by ognyandim** [ebullient/obsidian-slides-extended#54] [Q&A] [answered]
Hi,
I searched for Slides in the Obisidan community plugin browser and I stumbled upon this fork and the original project. 

I wonder how does this project build on or differ from Advanced Slides?

Is it because its support is discontinued? 
https://forum.obsidian.md/t/discontinued-advanced-slides-create-markdown-based-reveal-js-presentations-in-obsidian/28243

If this is the case is there any obstacle to continue the PRs in the original repo?

**Comment by SuzanaK** [comment]
You can't. Who will accept the PRs if the original developer won't accept them anymore?

**Comment by ebullient** [comment, marked as answer]
I do not go back to look for PRs on the previous repo. 

Any PRs related to processing/transforming markdown should be portable (and welcome), but I have made some changes: Slide Extended uses a newer version of reveal.js and fastify instead of express.
